modules/cnv-subscribing-to-hco-catalog.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/installing-container-native-virtualization.adoc
+
+[id="cnv-subscribing-to-hco-catalog_{context}"]
+= Subscribing to the KubeVirt HyperConverged Cluster Operator catalog
+
+You can subscribe to the *KubeVirt HyperConverged Cluster Operator* catalog from
+the {product-title} 4.1 web console. Subscribing gives the
+`kubevirt-hyperconverged` namespace access to the {ProductName} Operators.
```

Comment:
The reason why I think you need a short concept is that I got to this point and was like "why?"

maybe s/You can/Before you install {ProductName}.